59.md

```diff
@@ -0,0 +1,69 @@
+NIP-59
+======
+
+Gift Wrap
+-------------------------------
+
+`draft` `optional` `author:kieran`
+
+This NIP defines a simple method to protect event metadata leakage by using randomly generated keys.
+
+A gift wrap event is a `kind 1059` event which wraps another event with a single use key.
+
+This event kind can be used to wrap DM's or other events which are considered private.
+
+This is similar in concept to [Onion Routing](https://en.wikipedia.org/wiki/Onion_routing).
+
+To wrap an event, sign it as usual then serialize it to JSON. This string should then be encrypted using NIP-44 with a different private key. 
```

Comment:
Are we actually switching to nip 44? It's not yet merged, and I'm not sure it's a good idea. Although this would be a good place to start, gift wrap doesn't have to match nip 04 DMs

Reply:
I think it's a good place to start using it, though it does make it harder for people to support gift wrap, vs using the current nip-04 encryption.

I like NIP-44 simply because its versioned, we really need more things to be versioned